core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/common/WorkflowActor.scala

```diff
@@ -0,0 +1,19 @@
+package edu.uci.ics.amber.engine.architecture.common
+
+import akka.actor.{Actor, ActorLogging, Stash}
+import com.softwaremill.macwire.wire
+import edu.uci.ics.amber.engine.architecture.messaginglayer.MessagingManager
+import edu.uci.ics.amber.engine.architecture.receivesemantics.FIFOAccessPort
+import edu.uci.ics.amber.engine.common.ambermessage.WorkerMessage.DataMessage
+
+class WorkflowActor extends Actor with ActorLogging with Stash {
+
+  lazy val input: FIFOAccessPort = wire[FIFOAccessPort]
+  lazy val messagingManager: MessagingManager = wire[MessagingManager]
+
+  override def receive: Receive = {
```

Comment:
I saw `Processor` is changed to extend this class, I assume `Processor` also override the `receive` function so this function is not actually called. In this case, I suggest either remove this function since it's not used, or add clear comment saying it's not used right now..

Reply:
agreed. I will added a comment.